src/components/Button/Button.tsx

```diff
@@ -1,28 +1,40 @@
 import './Button.css';
 
+import { FC } from 'react';
 import classNames from 'classnames';
 
-const Button = ({
+interface ButtonProps {
+  children: string;
```

Comment:
1. `FC` already has types for `children`, so no need to put it here. 
2. `children` are `React.Element` AFAIK, not `text`

Reply:
1. Good to know, thanks. I read afterwards that it's a good idea to make it explicit, when a component is accepting children (as some components don't). Thus it might be good to leave it. Do you agree?

2. In this Button case I actually only want text as children. Is string ok then because it is more specific?